openfga_sdk/rest.py

```
@@ -278,6 +278,9 @@ async def handle_response_exception(
         if 200 <= response.status <= 299:
             return
 
+        if isinstance(response, aiohttp.ClientResponse) and not hasattr(response, "data"):
+            response.data  = await response.read()
```

Flake8 (7.2.0)

[error] multiple spaces before operator

(E221)